templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Driver?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <input type="submit" value="Yes" class="btn btn-danger">
```

Comment:
Add a `cancel` button here as well according to the [checklist](https://github.com/alecilchenko/py-taxi-service-user-form-and-validation/blob/develop/checklist.md#code-style). It should redirect to the previous page visited by the user.